src/app.module.ts

```diff
@@ -1,16 +1,25 @@
 import { Module } from '@nestjs/common';
+import { SurveyController } from "./survey/survey.controller";
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { AccountModule } from './account/account.module';
 import { AccountController } from './account/account.controller';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { DbConfig } from './config/db.config';
+import { SurveyService } from './survey/survey.service';
+import { SurveyModule } from './survey/survey.module';
+import { SurveyRepository } from './survey/survey.repository';
 
 @Module({
   
   imports: [TypeOrmModule.forRoot(DbConfig),
-    AccountModule],
-  controllers: [AccountController],
+    AccountModule,
+    SurveyModule],
+  controllers: [
+    AccountController,
+    SurveyController,
+  ],
+  providers: [SurveyService,SurveyRepository],
```

Comment:
this one

Reply:
some error was there if i didn't put this as provider , don't know why